e2etest/zt_preserve_smb_properties_test.go

```diff
@@ -306,3 +306,34 @@ func TestProperties_SMBTimes(t *testing.T) {
 		"",
 	)
 }
+
+func TestProperties_EnsureContainerBehavior(t *testing.T) {
```

Comment:
is there any test we need to add for datalake? or is this just for smb permissions?